### ISSUE TITLE
playInBackground causes crash

### ISSUE THREAD

**Issue by hellogerard** [TheWidlarzGroup/react-native-video#418]
Hello,

After merging #381, which fixed all the callbacks, `playInBackground` is now causing a runtime exception (this crash is alluded to in that PR). To reproduce, add a true `playInBackground` attribute to the `<Video />` component, e.g.:

```js
<Video
  ...
  playInBackground
  ...
/>
```

in the file `example/index.ios.js`.

Thanks.

Screenshot of crash:

<img width="829" alt="screen shot 2016-12-13 at 2 00 09 pm" src="https://cloud.githubusercontent.com/assets/200488/21156705/d1da9250-c144-11e6-9a1f-941bb01ed7cb.png">

**Comment by Lepozepo** [comment]
+1

**Comment by larryranches** [comment]
I am also getting this error when setting the playInBackground prop. If you remove playInBackground prop totally from the Video component, it does work though. @jameshartt

**Comment by shahen94** [comment]
https://github.com/react-native-community/react-native-video/issues/421

It crashes on me even if i haven't playInBackground

**Comment by hellogerard** [comment]
> It crashes on me even if i haven't playInBackground

I have experienced this as well. Since I am not an iOS developer, I am not fully sure what's going on. But the module crashes on the same line in 2 cases, so it seems to be the same cause:

1. On iPhone 5/5c/5s (_regardless_ of iOS version)
1. On iPhone 6+, _if_ `playInBackground` is true

I've scoured the Apple documentation to see what changed between iPhone versions, but to no avail.
